conn.go

```diff
@@ -258,6 +261,29 @@ func (conn *Conn) Kick(ctx context.Context, tube string, bound int) (int64, erro
 	return int64(count), nil
 }
 
+// KickJob kicks the job with the provided ID in the specified tube. This function returns
+// an error if kicking fails.
+func (conn *Conn) KickJob(ctx context.Context, tube string, jobID uint64) error {
```

Comment:
To bring this method in line with the `bury()`, `delete()` and `release(), could you:

1. Unexport this method
2. Have it accept the `*Job` as a parameter, as opposed to the `tube` and `ID` separate
3. Call it `kick()`